include/deal.II/grid/grid_tools.h

```diff
@@ -1148,7 +1152,9 @@ namespace GridTools
     const std::vector<std::vector<Tensor<1, spacedim>>> &vertex_to_cell_centers,
     const typename MeshType<dim, spacedim>::active_cell_iterator &cell_hint =
       typename MeshType<dim, spacedim>::active_cell_iterator(),
-    const std::vector<bool> &marked_vertices = {});
+    const std::vector<bool> &                              marked_vertices = {},
+    const RTree<std::pair<Point<spacedim>, unsigned int>> &used_vertices_rtree =
+      RTree<std::pair<Point<spacedim>, unsigned int>>());
```

Comment:
This combination looks a bit inconsistent. :slightly_smiling_face: 
```suggestion
      {});
```